### ISSUE TITLE
Too many requests

### ISSUE THREAD

**Issue by mwargan** [dusterio/laravel-aws-worker#39]
I think I'm misunderstanding something, but it looks like my Queue is receiving a message every minute even if its empty. Looking at the CRON schedule, I guess that's expected. But this is eating up my free tier for SQS with over 867k requests in two weeks. How do I lower the amount of messages sent to SQS, or stop a message being sent if its empty?

**Comment by dusterio** [comment]
there is an internal process (a daemon) running in the worker image - it's called sqsd. it's responsible for pulling from the queue, it's probably somewhere in the settings - you should be able to specify how often to poll

**Comment by mwargan** [comment]
@dusterio hmm - took a look at AWS Beanstalk config settings and I couldn't find anything that'd allow me to specify how often to poll. So are you saying this is unrelated to the CRON schedule? Or even this awesome package?

I did find [Long-Polling on the AWS docs](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-configure-long-polling-for-queue.html) using Receive Message Wait Time setting - is that what I'm looking for?

I just want to clarify if this is expected behaviour - I've set up the package according to your ReadMe, and it works great, it's just the frequency of requests that are unnecessary for me at this stage. Is it expected to have a request to SQS every minute or have I screwed up some config somewhere?
